src/index.html

```diff
@@ -7,7 +7,48 @@
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="body">
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+    <div class="stars stars--1">
+      <div class="stars__star stars__star--active"></div>
```

Comment:
Redundant classes, need to remove it everywhere
```suggestion
      <div class="stars__star"></div>
```

Reply:
@volodymyr-soltys97 In the description I have written that I have some issue with deploying the page. When I deployed this first time the stars appeared. Following that I have updated some code formatting and deployed the page again only to see that the stars do not appear anymore. Could you help me?